tests/spec/w3c/headers-spec.js

```diff
@@ -179,6 +179,88 @@ describe("W3C — Headers", function() {
     });
   });
 
+  describe("formerEditors", () => {
+    it("takes no former editor into account", async () => {
+      const ops = makeStandardOps();
+      const newProps = {
+        specStatus: "REC",
+        formerEditors: [],
+      };
+      Object.assign(ops.config, newProps);
+      const doc = await makeRSDoc(ops);
+
+      expect($("dt:contains('Former Editors:')", doc).length).toEqual(0);
+      expect($("dt:contains('Former Editor:')", doc).length).toEqual(0);
+    });
+
+    it("takes a single former editor into account", async () => {
+      const ops = makeStandardOps();
+      const newProps = {
+      specStatus: "REC",
+        formerEditors: [{
+          name: "NAME",
+          url: "http://URI",
+          company: "COMPANY",
+          companyURL: "http://COMPANY",
+          mailto: "EMAIL",
+          w3cid: "1234",
+        }],
+      };
+      Object.assign(ops.config, newProps);
+      const doc = await makeRSDoc(ops);
+
+      expect($("dt:contains('Former Editors:')", doc).length).toEqual(0);
+
```

Comment:
Sorry 😐 we are trying not to use jQuery in the project any more. Could you change these to just use querySelectorAll().

Reply:
Oh. I thought we could use jQuery in tests.
No problem. I'll update this.

Reply:
It actually turns out to be bad in tests too, as it adds a layer of indirection that makes things harder to debug when things go wrong.

Reply:
I'm thinking of adding these helper functions to `specHelper.js`: https://gist.github.com/sidvishnoi/2614f24af7d11051451a6a244bcecefb

Please do comment on that.

Reply:
Sorry, I’m not clear which helper functions you mean. Could you clarify?

Reply:
As javascript doesn't have one-liners for selectors like `dt:contains(someText)` and `el.next("dd")`, I'm writing the helper functions

For example, `selectorContains(selector, text, context = document)` equivalent to jQuery's `:contains` selector

I've added them in gist above.

Unless we plan to use some other selector library like Sizzle or create all the helper functions as we need them.

Reply:
I could be convinced, but I’ve personally not found a need for them.

Reply:
Can you suggest alternative to the following? 
``` js
expect($("dt:contains('Former Editors:')", doc).length).toEqual(0);
```

Reply:
So, a proposal elsewhere was to bring in Sizzle as a starting point. I’d be open to that - but would only want us to use it on the rare case the web platform can’t do something (e.g. “contains”)

Reply:
In the test files, I've seen a huge usage of `:contains` and `el.next(tagname)`. Most of selectors are supported with `querySelector[All]`. 

So what do you suggest for the test cases for this module? Should I continue to use jQuery like selectors until we finalise on this? Or create helpers?

Reply:
I will create a list of helper functions we may need that can help in making a decision whether to use some library like Sizzle or create our own (if a few).

Reply:
Sounds great!

Reply:
About suggestion from above:

const eds= [...doc.querySelectorAll(“dt”)].find({textContent}) => textContent.trim() === “Former Editors:”);

Or some variation of the above.

Reply:
gets a little verbose and repetitive though

Reply:
Yes, sometimes it definitely does. For the tests at least, the most important thing is limiting indirection (even at the cost of some repetition). But we make exceptions, like the makeRSDoc() function.